### PR TITLE
Clean up & optimize

### PR DESCRIPTION
**Pull request by ISSOtm**
This also fixes a bug where the joypad state wouldn't be sent over via the link cable.

### CODE REVIEW

File: main.asm [hunk 1]
```diff
@@ -1,166 +1,185 @@
...
 
-SECTION "romheader",ROM0[$100]
-	nop
-	jp Start
+VBlank: ; VBlank handler here
+    call Joypad
+    call DisplayJoypadState
+    ldh a, [hSendByLinkCable]
+    rra ; Bit 0 into carry
+    ldh a, [hCurrentJoypadState]
```
Comment (ISSOtm) [inline main.asm:47]: This is the link cable sending fix.

Comment (MineRobber9000) [reply main.asm:47]: What exactly does this fix? It seems to just do what is already being done, but more optimized.

Comment (MineRobber9000) [reply main.asm:47]: NVM, saw your post in #tools.

**Comment by MineRobber9000** [comment]
Thanks!

**Pull request merged**